content/ar/docs/home/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+approvers:
+- mboukhalfa
```

Comment:
```suggestion
# approvers:
# - chenopis
```

Please do not change the names of reviewers or approvers in individual documents.
Instead, please disable the section by toggling a line comment.